managers/restore-manager/RestoreStatusPoller.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const _ = require('lodash');
+const Promise = require('bluebird');
+const eventmesh = require('../../data-access-layer/eventmesh');
+const CONST = require('../../common/constants');
+const logger = require('../../common/logger');
+const catalog = require('../../common/models').catalog;
+const config = require('../../common/config');
+const RestoreService = require('./');
+const utils = require('../../common/utils');
+const EventLogInterceptor = require('../../common/EventLogInterceptor');
+const BaseStatusPoller = require('../BaseStatusPoller');
+
+class RestoreStatusPoller extends BaseStatusPoller {
+  constructor() {
+    super({
+      resourceGroup: CONST.APISERVER.RESOURCE_GROUPS.RESTORE,
+      resourceType: CONST.APISERVER.RESOURCE_TYPES.DEFAULT_RESTORE,
+      validStateList: [CONST.APISERVER.RESOURCE_STATE.IN_PROGRESS],
```

Comment:
just like backup , shouldn't we check for aborting state here as well ?

Reply:
For only `in_progress` state we are starting pollers. @muralisc can you please confirm

Reply:
We need to add polling support for restore abort similar to backup.